xml/cap_admin_upgrade.xml

```diff
@@ -131,51 +131,596 @@
    <xref linkend="sec-cap-skipped-release"/>.
   </para>
 
-<screen>&prompt.user;helm list</screen>
+  <para>
+   For upgrades from &productname; 1.5 to 1.5.1, if
+   the <literal>mysql</literal> roles of <literal>uaa</literal> and
+   <literal>scf</literal> are in high availability mode, it is recommended to
+   first scale them to single availability. Performing an upgrade with these
+   roles in HA mode runs the risk of encountering potential database migration
+   failures, which will lead to pods not starting.
+  </para>
 
   <para>
-   Begin by performing an upgrade of <literal>uaa</literal>.
+   The following sections outline the upgrade process for &productname;
+   deployments described by the given scenarios:
   </para>
 
-<screen>&prompt.user;helm upgrade <replaceable>susecf-uaa</replaceable> suse/uaa \
+  <itemizedlist>
+   <listitem>
+    <para>
+     The <literal>mysql</literal> roles are in high availability mode by setting
```

Comment:
Slight change in wording - "The <literal>mysql</literal> roles can be configured to run in high availability mode..."

Reply:
Wouldn't the suggested wording not really fit the context here since we're trying say something like

If your current setup is A, go to link X
If your current setup is B, go to link Y

Reply:
I don't feel strongly about it so feel free to disregard...just sounds like something's missing. Would "The mysql roles are _running_ in high availability mode..." sound better?

Reply:
Updated to use "are currently running in high availability mode"